README.md

```
@@ -0,0 +1,293 @@
+<img width="1270" height="1338" alt="스크린샷 2025-08-29 151320" src="https://github.com/user-attachments/assets/aa151682-f0b8-4905-a5ee-a6e61c15f050" />
+
+
+<br>
+
+## 7. Directory Structure
+<h3> 
+<details>
+<summary><a></a>Frontend</summary>
+<div markdown="1">
+  
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)